exercise1/exercise1.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Exercise 1</title>
+  </head>
+  <style>
+.company_img {
+  margin: 132px 0px 0px 255px;
```

Comment:
I think we should use the outer box and align it with some margin/padding rather than giving such huge margins to each object. Are you getting it?